app/views/search/_granule_search.html.erb

```diff
@@ -1,97 +1,116 @@
 <!-- ko with: project.searchGranulesCollection() -->
 <div class="master-overlay-content" data-master-overlay-pad="50">
+  <!--ko ifnot: cwic()-->
   <!-- ko if: cmrGranuleQuery.excludedGranules().length == 1 -->
   <p class="message message-warning">
     1 granule has been removed from your results.
     <a data-bind="click: $root.ui.collectionsList.focused().clearExclusions" href="#">Add it back</a>
   </p>
   <!-- /ko -->
-  <!-- ko if: cmrGranuleQuery.excludedGranules().length > 1 -->
-  <p class="message message-warning">
-    <span data-bind="text: cmrGranuleQuery.excludedGranules().length"></span> granules have been removed from your results.
-    <a data-bind="click: $root.ui.collectionsList.focused().clearExclusions" href="#">Add them back</a>
-  </p>
+    <!-- ko if: cmrGranuleQuery.excludedGranules().length > 1 -->
+    <p class="message message-warning">
+      <span data-bind="text: cmrGranuleQuery.excludedGranules().length"></span> granules have been removed from your results.
+      <a data-bind="click: $root.ui.collectionsList.focused().clearExclusions" href="#">Add them back</a>
+    </p>
+    <!-- /ko -->
   <!-- /ko -->
   <dl>
-    <dt>Granule ID</dt>
-    <dd class="granule-id-search search-single">
-      <section class="single-granule-id-search">
-        <input data-bind="value: cmrGranuleQuery.granuleIds" id="granule_id" type="text" placeholder="Search Granule IDs..." />
-        or
-        <a href="#" class="search-granules-multiple button-small button-outline-light">
-          Search Multiple <i class="fa fa-arrow-circle-o-right"></i>
-        </a>
-      </section>
-      <section class="multiple-granule-id-search">
-        <p>
-          <a href="#" class="search-granules-one button-small button-outline-light">
-            <i class="fa fa-arrow-circle-o-left"></i> Search One
+    <!--ko ifnot: cwic()-->
```

Comment:
I'd really like to find a better way to do this, long-term. The whole panel could/should probably be swapped.